platooning_strategic_IHP/src/platoon_strategic_ihp.cpp

```diff
@@ -173,16 +174,19 @@ namespace platoon_strategic_ihp
     bool PlatoonStrategicIHPPlugin::isVehicleRightInFront(double downtrack) 
```

Comment:
Now that we have cut-in capability, I think both isVehicleRightInFront() and isVehicleRightBehind() need to test CTD also to ensure that host is in the same lane as the platoon.  This allows the simple front or rear join.  If in different lanes, they will need to go through the cut-in logic, yes?